source/prerequisites/my-sql.txt

```diff
@@ -10,4 +10,92 @@ MySQL Migration Prerequisites
    :depth: 1
    :class: singlecol
 
-MySQL.
\ No newline at end of file
+To run sync jobs from a MySQL source database, the database may require 
+some configuration changes. If Relational Migrator determines the 
+database needs configuration changes, it automatically generates a 
+SQL script with the required changes. It is recommended to have a 
+Database Administrator (DBA) review the commands in this script and 
+perform their execution on the database server. This topic 
+provides more details on the required configuration steps.
+
+.. include:: /includes/fact-short-sync-job-desc.rst
+
+For details on supported versions of MySQL, see 
+:ref:`supported-databases`.
+
+About this Task
+---------------
+
+Connecting Relational Migrator to MySQL 
```

Comment:
The binlog is only needed for CDC jobs. Can we structure this topic similarly to the SQL Server one so that it's clear which prerequisites apply to each sync job type. Snapshot jobs have no prerequisite.